assets/scripts/libraries/cookies-opt-in.js

```diff
@@ -135,8 +135,8 @@ var docCookies = {
     // Set the default cookies. This JSON Object is saved as the cookie, but we use `initial_cookie_preferences` to maintain structure and various sanity checks
     var cookie_preferences = {
         essentials: true,
```

Comment:
This is in existing code - not related to your change. It looks like it's referred to as `essentials` here and as `essential` on line 122. Does this work?

Reply:
The `essential` on line 122 is used as a reference to the settings for the cookie preferences page and the `essentials` on line 137 is just the default cookie settings. It should still work having tested on local.